CMakeLists.txt

```diff
@@ -3,6 +3,9 @@ set(CMAKE_CXX_STANDARD 14)
 
 project(client)
 
+include(FeatureSummary)
+feature_summary(WHAT REQUIRED_PACKAGES_NOT_FOUND FATAL_ON_MISSING_REQUIRED_PACKAGES)
```

Comment:
Remove this instance of feature_summary, you need to call it only once at the end.